includes/amp-helper-functions.php

```diff
@@ -1943,3 +1905,26 @@ function amp_generate_script_hash( $script ) {
 	);
 	return 'sha384-' . $hash;
 }
+
+/**
+ * Get the AMP version of a URL.
+ *
+ * @since 2.1
+ *
+ * @param string $url URL.
+ * @return string AMP URL.
+ */
+function amp_get_url( $url ) {
+	return add_query_arg( amp_get_slug(), 1, $url );
+}
+
+/**
+ * Determine whether the current request has the AMP query parameter set with the required value '1'.
+ *
+ * @since 2.1
+ *
+ * @return bool True if the AMP query parameter is set with the required value, false if not.
+ */
+function amp_has_query_var() {
+	return isset( $_GET[ amp_get_slug() ] ) && 1 === filter_var( $_GET[ amp_get_slug() ], FILTER_VALIDATE_INT ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
I don't think the value should need to be _required_ to be `1`. It is just `1` as a generic non-empty value.

```suggestion
	return isset( $_GET[ amp_get_slug() ] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```